plugins/labhub.py

```diff
@@ -387,3 +388,65 @@ def pr_stats(self, msg, match):
                     state=type(self).community_state(pr_count)
                  )
         yield reply
+
+    @re_botcmd(pattern=r'pr\s+list\s+([^/]+)',  # Ignore LineLengthBear, PyCodeStyleBear
+               re_cmd_name_help='pr list <repo>',
+               flags=re.IGNORECASE)
+    def pr_list(self, msg, match):
+        """List PRs ready to be merged."""  # Ignore QuotesBear
+        repo_name = match.groups(1)[0]
+
+        try:
+            mrs = self.REPOS[repo_name].merge_requests()
+        except KeyError:
+            return 'Repository doesn\'t exist.'
+        else:
+            checks = []
+
+            def check_mr(function):
+                checks.append(function)
+                return function
+
+            @check_mr
+            def check_labels(mr):
+                labels = mr.labels
+                rl = ['process/approved', 'process/pending review',
```

Comment:
these belong in config somewhere ; at least put them in a CONSTANT at the top of this module.